modules/uptimes.sh

```diff
@@ -1,24 +1,26 @@
 #!/bin/bash
- SCRIPTPATH=$(realpath "$0")
- SCRIPTFOLDER=$(dirname "$SCRIPTPATH")
 
 function uptimes(){
-if [[ ! -f "$SCRIPTFOLDER/templates/up_time" ]]
-then
-uptime -s > "$SCRIPTFOLDER"/templates/up_time 
-up=$(uptime -s)
-echo "System is up since $up"
-else
-last_up_time=$(cat "$SCRIPTFOLDER"/templates/up_time)
-curr_up_time=$(uptime -s)
-if [[ "$last_up_time" != "$curr_up_time" ]]
-then
-uptime -s > "$SCRIPTFOLDER"/templates/up_time
-echo "Down Time was between $last_up_time and $curr_up_time"
-echo "System has been up since $curr_up_time"
-else
-up=$(uptime -s)
-echo "System is up since $up"
-fi
-fi 
+  last_up_time=$(cat "$SCRIPTFOLDER"/templates/up_time)
+  curr_up_time=$(uptime -s)
+  if [[ "$last_up_time" != "$curr_up_time" ]]
+  then
+    uptime -s > "$SCRIPTFOLDER"/templates/up_time
+    echo "Down Time was between $last_up_time and $curr_up_time"
+    echo "System has been up since $curr_up_time"
+  else
+    up=$(uptime -s)
+    echo "System has been up since $up"
+  fi
+}
+
+function uptimes_help(){
+  echo ""
+  echo "Usage: $(basename "$0") uptimes"
+  echo ""
+  echo "Example:"
+  echo " $(basename "$0") uptimes"
+  echo "    This will display since when the system is running"
+  echo "    If the system was down it will also display the time range between which the system was down"
```

Comment:
Almost done!

Grammar changes:
`This will display since when the system is running` and `If the system was down it will also display the time range between which the system was down` 
can be changed to `displays the system uptime and time range between which it was down`

Reply:
hi made the changes and i apologize for my grammar and weak command over language pushing the changes now

Reply:
making the changes now thanks for the review and i apologize for my the trivial errors

Reply:
for the trivial errors*